src/lib.rs

```diff
@@ -71,8 +71,18 @@ use serde::{de::DeserializeOwned, Serialize};
 use std::fmt;
 use std::sync::Arc;
 
+/// Size of an initialisation vector.
+pub const IV_SIZE: usize = box_::NONCEBYTES;
+/// Size of a secret symmetric key.
+pub const SYMMETRIC_KEY_SIZE: usize = secretbox::KEYBYTES;
+/// Size of a public signing key.
+pub const PUBLIC_SIGN_KEY_SIZE: usize = 32;
+
 /// Represents public signature key.
-pub type PublicSignKey = [u8; 32];
+pub type PublicSignKey = [u8; PUBLIC_SIGN_KEY_SIZE];
+
+/// Initialisation vector.
+pub type Iv = [u8; IV_SIZE];
```

Comment:
Maybe have a longer name for a public type? e.g. `InitVec`, `InitializationVec`, etc.